GoMeet/src/main/webapp/test/spec/spec-meeting-event-dao.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * Creates <input> elements with the names and values as given in arr.
+ * @param {Array} arr Array of objects each in the format {name: value},
+ * representing the name and value of the <input> elements to create.
+ * @returns {Array} Array of <input> elements created
+ */
+function createInputElems(arr) {
+  var inputElements = [];
+  for (let i = 0; i < arr.length; i++) {
+    var elem = document.createElement('input');
```

Comment:
nit: avoid using var. Style guide: "Declare all local variables with either const or let,  use const by default, unless a variable needs to be reassigned."